client.go

```diff
@@ -33,6 +34,7 @@ func (c *Client) listen() {
 		case arbor.NewMessageType:
 			if c.recieveHandler != nil {
 				c.recieveHandler(m.ChatMessage)
+				beeep.Notify("Muscadine", m.ChatMessage.Content, "")
```

Comment:
You could do something like this:
```suggestion
if c.Composer.Username != m.ChatMessage.Username {
	beeep.Notify("Muscadine", m.ChatMessage.Content, "")
}
```
If you choose to "accept" this suggested change, please fix the indentation afterward. Github mangled it.